### PR TITLE
feat(readonly): listen on readonly attribute

### PR DESCRIPTION
**Pull request by IcanDivideBy0**

### CODE REVIEW

**Comment by hugo-agbonon** [comment]
k

File: test/angular-ckeditor.js [hunk 1]
```diff
@@ -157,4 +157,29 @@ describe('CKEditor directive', function () {
...
+      scope.content = 'Hello';
+      scope.onReady = done;
+      scope.readonly = false;
+
+      var element = $compile(
+        '<div contenteditable="true" ckeditor readonly="{{readonly}}" ng-model="content" ready="onReady()"></div>'
+      )(scope);
+    });
+
+    it('should observer readonly attribute', function () {
```
Comment (hugo-agbonon) [inline test/angular-ckeditor.js]: observe [the] readonly attribute

**Comment by hugo-agbonon** [comment]
micro-fix et ok

File: test/angular-ckeditor.js [hunk 2]
```diff
@@ -157,4 +157,29 @@ describe('CKEditor directive', function () {
     });
   });
 
+  describe('readonly', function () {
+    beforeEach(function (done) {
+      scope.content = 'Hello';
+      scope.onReady = done;
+      scope.readonly = false;
```
Comment (Yvem) [inline test/angular-ckeditor.js]: le test serait encore meilleur si readonly était non défini au départ. Comme ça on testerait les 3 cas : undef / true / false

**Comment by Yvem** [comment]
micro-fix et :ok:

**Comment by Devqon** [comment]
This does not work as I hoped. I have an angular directive that goes through all form elements, and do this:

    if ($inputs.length != 0) {
        $inputs.each((index, elem) => {
            var $this = $(elem);

            // TODO: how to handle ckeditor
            $this.removeAttr("ng-disabled");
            $this.removeAttr("data-ng-disabled");
            $this.removeAttr("ng-readonly");
            $this.removeAttr("data-ng-readonly");
            $this.prop("disabled", true);
            $this.prop("readonly", true);
            $this.attr("disabled", "disabled");
            $this.attr("readonly", "readonly");
        });
    }

The angular-ckeditor however, does not see this change and does not apply the `setReadonly()`

**Pull request merged**